base_layer/p2p/src/proto/tari.p2p.liveness.rs

```diff
@@ -10,6 +10,9 @@ pub struct PingPongMessage {
     /// Metadata attached to the message. The int32 key SHOULD always be one of the keys in `MetadataKey`.
     #[prost(map = "int32, bytes", tag = "3")]
     pub metadata: ::std::collections::HashMap<i32, std::vec::Vec<u8>>,
+    /// Indicates the application version from which the message was sent
```

Comment:
You'll need to update the liveness.proto file which will generate this struct :)

Reply:
Thanks.